lms/djangoapps/grades/config/models.py

```diff
@@ -24,6 +24,19 @@ class PersistentGradesEnabledFlag(ConfigurationModel):
     feature to take effect.
 
     .. no_pii:
+
+    .. toggle_name: PersistentGradesEnabledFlag.enabled
+    .. toggle_implementation: ConfigurationModel
+    .. toggle_default: False
+    .. toggle_description: When enabled, grades are persisted. This means that PersistentCourseGrade objects are
+       created for student grades. In order for this to take effect, CoursePersistentGradesFlag objects must also be
+       created individually for each course. Alternatively, the PersistentGradesEnabledFlag.enabled_for_all_courses
+       waffle flag or the PERSISTENT_GRADES_ENABLED_FOR_ALL_TESTS feature flag can be set to True to enable this
+       feature for all courses.
+    .. toggle_use_cases:  open_edx
```

Comment:
Same here. This should be a temporary rollout use-case.  Our latest features will no longer work on non-persistent grades.

Reply:
:ok_hand: I'll switch to a `monitored_rollout` for now and will update this PR with the new annotation format in a later commit.